app/src/main/java/com/keylesspalace/tusky/db/AccountManager.kt

```diff
@@ -243,4 +265,14 @@ class AccountManager @Inject constructor(db: AppDatabase) {
 
         return accounts.size > 1 // "disambiguate"
     }
+
+    /**
+     * Clear drafts alert for current user
+     * Caller's responsibility to ensure there is a current user
+     */
+    fun clearDraftsAlert() {
```

Comment:
Instead of using `activeAccount!!.id` a few lines down, if you change this function so that it takes the account ID as a parameter, and don't allow null values for the parameter then it ensures the calling code does a null check.